src/adaptors/angle/index.js

```diff
@@ -0,0 +1,34 @@
+const { AppStream } = require('aws-sdk');
+const utils = require('../utils');
+
+const networks = {
+  1: 'Ethereum',
+  137: 'Polygon',
+};
+
+let apyData;
+const getPoolsData = async () => {
+  const apyData = await utils.getData('https://api.angle.money//v1/incentives');
+
+  const result = [];
+  for (const staking of Object.keys(apyData)) {
+    if (apyData[staking].deprecated) continue;
+    // the identifier is the voting gauge address and not the address of the staking pool
+    const pool = {
+      pool: apyData[staking]?.address, // address of the staking pool
+      chain: networks[apyData[staking]?.network] || 'Other',
+      project: 'angle',
+      symbol: apyData[staking]?.name,
```

Comment:
re symbol:

I have noticed that the symbols are like `"Uni-V2 agEUR/FEI LP"`. One thing we do on the server side is adding additional fields to each pool such as if its a stablecoin, if its single exposure etc. 
The functions adding that info use the symbol as an input arg. 
If we rely on `apyData[staking]?.name` then the output of those additional funcs will be wrong.

We can do 2 things here:
a) we change the symbol at the adaptor level
b) we add logic on the backend

if we go with a) we'd need to make the following change:

```
  symbol = apyData[staking]?.name.replace('/', '-').split(' ');
  // eg "Uni-V2 agEUR/FEI LP" keep -> 'agEUR/FEI' or "FRAX/EUR Perp" -> 'FRAX/EUR'
  symbol = symbol.length > 2 ? symbol[1] : symbol[0];
```
I know its not ideal cause we loose some info of the specific pool names but should still be good informative enough for the user to identify the correct pool. let me know what u think

Reply:
Thanks for the feedback! I fixed the two other issues. 

About the symbols, I'm not sure what is the best solution here. Basically, we have three types of staking right now: 
- staking for LP tokens
- staking for sanTokens (deposit USDC/DAI/..., get sanDAI/EUR that can be staked) -> always single exposure
- rewards distributed to users opening Perpetual positions from one of the collateral -> exposed to USD/EUR price changes

I feel like doing a) would make it clearer for #2 if it explains that it's a single exposure, but less so for #1 and #3 so probably not ideal. 

In any case, users will have to go through our frontend where we make an effort to explain each situation's context in great details, so you might not even need to add it yourself. (if you find anything unclear there please let me know btw)

Do you have an example of how this context looks like maybe?

Reply:
ty for the info on your side! One question: why is `sanDAI/EUR` token "single exposure"? if i deposit dai, shouldn't i also be exposed to agEUR in that case? 

--------------- 
Generally, let me try making things a big clearer regarding the toggles we have on the yield page:

<img width="546" alt="Screenshot 2022-05-25 at 12 34 42" src="https://user-images.githubusercontent.com/74533329/170252907-d74a5b46-269c-45da-8dc0-3c0b6656f661.png">

`Stablecoins`: A pool is exposed to stablecoins only eg. DAI on Aave, USDC-DAI on Uniswap etc.
`Single Exposure`: A pool is exposed to 1 token only. eg DAI on Aave, stETH on lido etc.
`No IL`: eg if pool consists of 1 token only or if more than 1 then they have to track the same underlying asset eg DAI-USDC-USDT on curve would be labelled with No IL (under the assumption of no depeg...given recent events I might actually change this, but thats at least what we do at the moment)

Lets map Angle's 3 different types of staking:
1) LP: `"Uni-V2 agEUR/FEI LP"` would be: `Stablecoin: True`, `Single Exposure: False`, `No IL: True` 
2) sanToken: `"sanFEI/EUR"`: would be: `Stablecoin: True`, `Single Exposure: False`, `No IL: True`
3) Perp: `"USDC/EUR Perp"` would be: `Stablecoin: True`, `Single Exposure: False`, `No IL: True`

Lmk if that makes sense and I got it right, especially for 2)

Reply:
You're welcome! 

When depositing DAI in Angle the sanDAI/EUR means that your DAI has been deposited in the pool over-collateralizing agEUR, as opposed to any other potential stablecoin we might launch. This DAI will be reinvested in Aave for example, but never pooled with agEUR. Thus, it is a single token exposure. 

I put in bold the two aspects that I think should be different: 
1. LP: `"Uni-V2 agEUR/FEI LP"` would be: `Stablecoin: True`, `Single Exposure: False`, `No IL: **False**` (as agEUR and FEI are pegged to two different values, EUR and USD)
2. sanToken: `"sanFEI/EUR"` would be: `Stablecoin: True`, `Single Exposure: **True**`, `No IL: True`
3. Perp: `"USDC/EUR Perp"` would be: `Stablecoin: True`, `Single Exposure: False`, `No IL: True`

On 3), Perp are not necessarily based on stablecoins. wETH/EUR Perp for example would be `Stablecoin: False`, as collateral used is wETH.

Reply:
got it. ty for explaining!

re the second section, yes that makes sense. regarding 3) thats ok, we will handle those cases so no worries there.

I think we are at the same page now. Which brings us back to how we could handle this on the adaptor side:

```
- remove substrings before/after empty space like Uni-v2, LP, Perp
- replace `/` with `-`
- and for symbols including `san` i would keep only the deposit token eg `sanFEI/EUR` -> `FEI`
```

if we change it like that then everything would integrate nicely with the backend.

Downside of this approach is that we loose a bit of info on the symbols/pool names. i know its not ideal, we are doing similar things for other projects as well. also, i'm actually considering to change this in the near future to have the full poolname as well. 
for now though, i don't think its a big problem if we do the apply the suggested steps, cause the user will check out your site anyways. But let me know if u disagree):

Reply:
Great! 

About doing the change, I think it's fine for 1) and 2), as you probably have other LP staking pools listed so users should understand. 

However, I think it would get really confusing for 3) as users could easily take these for liquidity pools or deposits whereas it is a very different mechanism. 

Maybe we can try to split the cases at the adaptor level to fix this for 1) and 2) and leave the full `Perp` names?
If splitting isn't too heavy code wise let's go for this :)